### ISSUE TITLE
Gitg can not start

### ISSUE THREAD

**Issue by biji** [alexlarsson/gnome-apps-nightly#29]
with this error:
*\* (gitg:2): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-4vcZlXBWSj: Connection refused
Gtk-Message: Failed to load module "pk-gtk-module"
Gtk-Message: Failed to load module "pk-gtk-module"
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown

thanks

**Comment by biji** [comment]
would you please rebuild in the repo
thanks
